script/new-file.txt

```diff
@@ -1 +1,3 @@
-Adding another file in feature-1 branch
\ No newline at end of file
+Adding another file in feature-1 branch
```

Comment:
That's a great job including this code here

Reply:
thank you for providing your feedback